mlflow/utils/uri.py

```diff
@@ -281,3 +282,35 @@ def dbfs_hdfs_uri_to_fuse_path(dbfs_uri):
         )
 
     return _DBFS_FUSE_PREFIX + dbfs_uri[len(_DBFS_HDFS_URI_PREFIX) :]
+
+
+def resolve_uri_if_local(local_uri):
+    """
+    if `local_uri` is passed in as a relative local path, this function
+    resolves it to absolute path relative to current working directory.
+
+    :param local_uri: Relative or absolute path or local file uri
+
+    :return: a fully-formed absolute uri path
```

Comment:
This return docstring doesn't quite seem accurate. It's either a URI with an absolute path component or an absolute filesystem path.